src/interfaces.ts

```diff
@@ -333,6 +337,15 @@ export interface Web3AuthOptions {
    * only scalar will be exported, scalar can be used for signing outside of this sdk but not for importing the key in other wallets.
    */
   useClientGeneratedTSSKey?: boolean;
+
+  /**
+   * @defaultValue `false`
+   * Set this flag to true to use the legacy flag for signing
+   * legacy flag do not support multicurve mode
+   * legacy ed25519 customAuth is only supported in legacy mode
+   * Note: This option is set to false by default.
+   */
```

Comment:
```suggestion
  /**
   * @defaultValue `false`
   * Set this flag to true to use the legacy mode.
   * Legacy mode does not support multi-curve.
   * CustomAuth with key type ed25519 is only supported in legacy mode.
   * Note: This option is set to false by default.
   */
```
I like that this has a comment, but it's a bit raw. could add some punctuation and revise language.